### PR TITLE
Some new 1.14.3 mappings

### PR DESCRIPTION
**Pull request by NeunEinser**

### CODE REVIEW

File: mappings/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS apo net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.mapping:1]: Just to make sure, does this match the registry name?

File: mappings/net/minecraft/resource/JsonResourceLoader.mapping [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CLASS xg net/minecraft/resource/JsonResourceLoader
+	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD b FILE_TYPE_LENGTH I
+	FIELD c gson Lcom/google/gson/Gson;
+	FIELD d category Ljava/lang/String;
```
Comment (liach) [inline mappings/net/minecraft/resource/JsonResourceLoader.mapping]: mind describing this class and this field?

File: mappings/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.mapping:1]: Yes, it is.

File: mappings/net/minecraft/resource/JsonResourceLoader.mapping [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CLASS xg net/minecraft/resource/JsonResourceLoader
```
Comment (liach) [inline mappings/net/minecraft/resource/JsonResourceLoader.mapping]: I'd prefer `JsonDataLoader` as this is only used by data.

File: mappings/net/minecraft/resource/JsonResourceLoader.mapping [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD b FILE_TYPE_LENGTH I
```
Comment (liach) [inline mappings/net/minecraft/resource/JsonResourceLoader.mapping]: `FILE_SUFFIX_LENGTH`

File: mappings/net/minecraft/resource/JsonResourceLoader.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/resource/JsonResourceLoader.mapping]: `resourceType` or `dataType`

File: mappings/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.mapping:1]: Yes, it does.

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/resource/JsonResourceLoader.mapping (hunk 3 above)
Comment (NeunEinser) [reply mappings/net/minecraft/resource/JsonResourceLoader.mapping]: Yeah, I wanted to somehow squeeze `Data` in there, but most classes in that package have `Resource` somewhere in there name (which makes sense, in that context), but I agree that `Data` is better.
It also seems to be only intended for data, as all the strings in that class all talk about data.

Comment (asiekierka) [reply mappings/net/minecraft/resource/JsonResourceLoader.mapping]: It's fine IMO

File: mappings/net/minecraft/client/gui/screen/TitleScreen.mapping [hunk 5]
```diff
@@ -1,8 +1,30 @@
...
-	FIELD a panoramaCubeMap Ldkg;
-	FIELD b panoramaOverlay Lqs;
+	CLASS czy$a Warning
+		FIELD b line2Width I
+		FIELD c startX I
+		FIELD d startY I
+		FIELD e endX I
+		FIELD f endY I
+		FIELD g line1 Ljm;
+		FIELD h line2 Ljm;
```
Comment (liach) [inline mappings/net/minecraft/client/gui/screen/TitleScreen.mapping:9]: `firstLine`, `secondLine`

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/client/gui/screen/TitleScreen.mapping (hunk 5 above)
Comment (NeunEinser) [reply mappings/net/minecraft/client/gui/screen/TitleScreen.mapping:9]: Sorry for the late reply, I am currently experiencing an internet outage at home.
The translation strings read `.line1.` and `.line2.`. If first and second is preferred nonetheless, I can change it tomorrow.

Comment (liach) [reply mappings/net/minecraft/client/gui/screen/TitleScreen.mapping:9]: this should be fine. no worries.

**Review by asiekierka — APPROVED** [review]

**Pull request merged**